Sources/AblyChat/Messages.swift

```diff
@@ -0,0 +1,74 @@
+import Ably
+
+public protocol Messages: AnyObject, Sendable, EmitsDiscontinuities {
+    func subscribe(bufferingPolicy: BufferingPolicy) -> MessageSubscription
+    func get(options: QueryOptions) async throws -> any PaginatedResult<Message>
+    func send(params: SendMessageParams) async throws -> Message
+    var channel: ARTRealtimeChannelProtocol { get }
+}
+
+public struct SendMessageParams: Sendable {
+    public var text: String
+    public var metadata: MessageMetadata?
+    public var headers: MessageHeaders?
+
+    public init(text: String, metadata: MessageMetadata? = nil, headers: MessageHeaders? = nil) {
+        self.text = text
+        self.metadata = metadata
+        self.headers = headers
+    }
+}
+
+public struct QueryOptions: Sendable {
+    public enum Direction: Sendable {
```

Comment:
I left a comment in an ADR [here](https://ably.atlassian.net/wiki/spaces/CHA/pages/3126362123/CHADR-031+Chat+SDK+API+Design+Review+for+Milestone+1#Chat-Messages-History) about this but TLDR... 

I think having a direction with forwards or backwards in confusing when talking about Chat/Messages. 

This should be sortby: .newestFirst or .oldestFirst imo, wdyt?

Reply:
I agree that the name is confusing but it sounds like this is an issue with the JS API and should hence be fixed there first. I think the only divergences from JS in this PR should be ones that are for adapting the API to Swift.

What was the outcome of your flagging this confusing naming in the ADR?

Reply:
I'm not sure I feel as strongly about not diverging from the JS API. e.g. it's rare that a company utilising our chat SDKs will have the same developer implementing JS, Kotlin, and Swift.

Fixing everything in JS first causes that to be the bottleneck, I think we should be able to implement improvements in our SDKs (with approval from the JS team) and they can play catch up if needed.

This might be worth syncing with Andy on to see what approach they'd rather. I'll raise this in standup and see what we think first